docusaurus/docs/dev-docs/quick-start.md

```diff
@@ -166,71 +183,55 @@ You have just created a basic data structure for your Strapi project! You can ke
 
-<details style={detailsStyle}>
-<summary style={summaryStyle}>Step 1: Host the code of your Strapi project on GitHub</summary>
+To deploy your project on Strapi Cloud, in your terminal:
 
-### Step 1: Host the code of your Strapi project on GitHub
+1. If the server for your local Strapi project is running, which should be the case if you followed this tutorial so far, press `Ctrl-C` to stop the server.
+2. Ensure you are in the folder of your Strapi project (if needed, run for instance `cd my-strapi-project` to reach this folder), and run the following command:
 
-Create a new GitHub repository and push the code of your Strapi project to this repository. If you're not already familiar with GitHub, the togglable content below should get you started 👇
+    <Tabs groupId="yarn-npm">
 
-<details>
-<summary>Steps required to push your Strapi project code to GitHub:</summary>
+    <TabItem value="yarn" label="Yarn">
 
-1. In the terminal, ensure you are still in the `my-project` folder that hosts the Strapi project we created. If you followed this tutorial closely so far, we should still be there.
-2. Run the `git init` command to initialize git for this folder.
-3. Run the `git add .` command to add all modified files to the git index.
-4. Run the `git commit -m "Initial commit"` command to create a commit with all the added changes.
-5. Log in into your GitHub account and [create a new repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/quickstart-for-repositories). Give the new repository a name, for instance `my-first-strapi-project`, and remember this name.
-6. Go back to the terminal and push your local repository to GitHub:
+      ```sh
+      yarn strapi deploy
+      ```
 
-  a. Run a command similar to the following: `git remote add origin git@github.com:yourname/my-first-strapi-project.git`, ensuring you replace `yourname` by your actual GitHub profile name, and `my-first-strapi-project` by the actual name you used at step 4.
+    </TabItem>
 
-  b. Run the `git push --set-upstream origin main` command to finally push the commit to your GitHub repository.
+    <TabItem value="npm" label="NPM">
 
-Additional information about using git with the command line interface can be found in the [official GitHub documentation](https://docs.github.com/en/migrations/importing-source-code/using-the-command-line-to-import-source-code/adding-locally-hosted-code-to-github#adding-a-local-repository-to-github-using-git).
+      ```sh
+      npx @strapi/strapi deploy
+      ```
 
-</details>
+    </TabItem>
 
-</details>
-
-<details style={detailsStyle}>
-<summary style={summaryStyle}>Step 2: Create a Strapi Cloud account and a new Strapi Cloud project</summary>
-
-### Step 2: Create a Strapi Cloud account and a new Strapi Cloud project
+    </Tabs>
 
-To create a new Strapi Cloud account:
+Within a few moments, the script will get your local project hosted on Strapi Cloud. 🚀 
```

Comment:
the `script` or the `command`? The term is never used before, I was a bit surprised when reading it at first. It's just a comment, feel free to ignore it

Reply:
To me, the command executes the script, but that's the script that builds/compresses/deploys the project 😊

Reply:
```suggestion
Within a few moments, your local project will be hosted on Strapi Cloud. 🚀 
```
We could just shorten it like this :)